src/amms/uniswap_v3/mod.rs

```diff
@@ -204,7 +205,14 @@ impl AutomatedMarketMaker for UniswapV3Pool {
                 self.liquidity = swap_event.liquidity;
                 self.tick = swap_event.tick.unchecked_into();
 
-                // tracing::debug!(?swap_event, address = ?self.address, sqrt_price = ?self.sqrt_price, liquidity = ?self.liquidity, tick = ?self.tick, "UniswapV3 swap event");
+                info!(
```

Comment:
This is going to be pretty verbose on sync right? Do we want this as an `info!()` over `debug!()`?

Reply:
Same with the rest of these

Reply:
The `UniswapV3Pool.sync()` method is only used when updating an already synced state space. Initial syncing via the `StateSpaceBuilder` does not call this method. Or are you referring to this log will be verbose when syncing recent state changes from `latest_block` to `chain_tip`?